.github/scripts/get-files-sha.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-echo sha=$(cd $1; git log -1 --pretty=format:"%H" ${@:2}) >> "$GITHUB_OUTPUT"
+echo $(cd $1; git log -1 --pretty=format:"%H" ${@:2})
```

Comment:
I guess echo and $() are not needed here anymore

Reply:
Fixed.